packages/studio-ui/src/web/components/Layout/BottomPanel/Debugger/views/Summary.tsx

```diff
@@ -37,6 +37,10 @@ export default class Summary extends React.Component<Props> {
       return null
     }
 
+    if (!this.props.event.processing?.completed) {
```

Comment:
**this is the fix**

everything else is about fixing react errors in chrome console or adding typings